app/views/medicaid/amounts_income/edit.html.erb

```diff
@@ -1,8 +1,12 @@
-<% content_for :header_title, "Income & Expense Amounts" %>
+<% content_for :header_title, "Income &amp; Expense Amounts".html_safe %>
```

Comment:
I think that this file will have merge conflicts with `master` because we recently updated it to have a slightly different style, which affected the input field names